internal/storage/storageutil/custom_retry.go

```diff
@@ -25,6 +27,10 @@ func ShouldRetry(err error) (b bool) {
 		return
 	}
 
+	if st, ok := status.FromError(err); ok && st.Code() == codes.ResourceExhausted {
```

Comment:
is this a temporary change?

Reply:
Yes, adding a todo to remove the change later.